app/Helpers/database/set-request.php

```diff
@@ -36,7 +36,12 @@ function getUserRequestList(?string $user = null): array
         GROUP BY
             sr.GameID
         ORDER BY
-            GameTitle ASC";
+            CASE
+                WHEN gd.Title LIKE '~%' THEN
+                    SUBSTRING(gd.Title, LOCATE(' ', gd.Title) + 1)
+                ELSE
+                    gd.Title
+            END ASC";
```

Comment:
does this lead to the same sorting order as before the unicode change?
for game lists it was changed to
```php
ifStatement("gd.Title LIKE '~%'", 1, 0) . ", gd.Title"
```

Reply:
It's slightly different. Is our intent to put all prefixed games (eg: Hack, Unlicensed, Homebrew) at the bottom of the list? If so, we can do:

```sql
        ORDER BY " . ifStatement("gd.Title LIKE '~%'", 1, 0) . ", gd.Title";
```

Reply:
at least that was the behaviour before if i'm no mistaken.
as soon as the tags are gone from the title and are added as tags they would sort the way you prepared it here.
we can go for that right away, too, but should probably change the other listings while at it?

oor, we remove all of that special behaviour and move those tags to the end of the title - like `[m]` achievements tags.

Reply:
Let's keep it consistent with the game page per your suggestion. Truthfully, I can't remember what the previous behavior was. Updated in https://github.com/RetroAchievements/RAWeb/pull/1977/commits/fdd071bc93feca2ac72485c9a85aec0e76cbd574.

Reply:
alright, let's go with that for now and discuss our desired sorting strategy for future reference.
sorting special characters to the beginning of the list is technically correct - that's how most file systems do it, too.